rclpy/test/test_action_graph.py

```diff
@@ -51,6 +51,21 @@ def setUpClass(cls):
         cls.action_server20 = ActionServer(cls.node2, Fibonacci, TEST_ACTION0, lambda: None)
         cls.action_server21 = ActionServer(cls.node2, Fibonacci, TEST_ACTION1, lambda: None)
 
+        assert cls.wait_for_node(node=cls.node1, remote_node=cls.node0, timeout=2)
+        assert cls.wait_for_node(node=cls.node1, remote_node=cls.node2, timeout=2)
+        assert cls.wait_for_node(node=cls.node2, remote_node=cls.node0, timeout=2)
+        assert cls.wait_for_node(node=cls.node2, remote_node=cls.node1, timeout=2)
+
+    @staticmethod
+    def wait_for_node(*, node, remote_node, timeout=float('+inf')):
```

Comment:
If the default value isn't used I would suggest to remove it.

Reply:
Fair enough. Removed in 1d9dc651a0fda1e7206af15e85c8f6a7973971ff.